src/client/gfs_client.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CLIENT_GFS_CLIENT_H
+#define CLIENT_GFS_CLIENT_H
+
+#include "google/protobuf/stubs/status.h"
+#include "google/protobuf/stubs/statusor.h"
+
+namespace gfs {
+namespace client {
+
+// Use the Status and Statusor objects provided by protobuf
+using google::protobuf::util::Status;
```

Comment:
c++ readability advice:

never use `using` (namespace alias) inside a header file, and move these to `.cc` explicitly for `Status` in `.cc`.

instead, use fully qualified `google::protobuf::util::Status` for all declarations in headers file

Reply:
Resolved